modules/cluster-node-tuning-operator-verify-profiles.adoc

```diff
@@ -28,3 +28,22 @@ worker-b         openshift-node            True      False      6h28m
 * `APPLIED`: `True` if the TuneD daemon applied the desired profile. (`True/False/Unknown`).
 * `DEGRADED`: `True` if any errors were reported during application of the TuneD profile (`True/False/Unknown`).
 * `AGE`: Time elapsed since the creation of Profile object.
+
+The ClusterOperator/node-tuning object also contains useful information about the Operator and its node agents' health.
+For example, operator misconfiguration will be reported via ClusterOperator/node-tuning status messages.
+
+[source,terminal]
+----
+$ oc get co/node-tuning -n openshift-cluster-node-tuning-operator
+----
+
+.Example output
+[source,terminal]
+----
+NAME          VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
+node-tuning   4.13.1    True        False         True       60m     1/5 Profiles with bootcmdline conflict
+----
+
+If either the ClusterOperator/node-tuning or a Profile object's status
```

Comment:
Can you remove the hard wrap? Our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines) recommend undoing it where possible.